src/CheckoutForm.js

```diff
@@ -2,45 +2,98 @@
 import React, {Component} from 'react';
 
 class CheckoutForm extends Component{
+
+    constructor(){
+        super()
+        this.state = {
+            fullName: '',
+            email: '',
+            address: '',
```

Comment:
1. Can we set address as an object instead of a string? I haven't defined the model yet, but it will contain firstName, lastName, addressLine1, addressLine2(optional), city, state, zip.

2. You may want to have separate shipping and billing address fields on state. In case the user does not enable the sameShipAddress option.

Reply:
+ 1 to @5t4mp5 said.